Libraries/src/Amazon.Lambda.Serialization.SystemTextJson/LambdaJsonSerializer.cs

```diff
@@ -0,0 +1,150 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using System.Text.Json;
+using Amazon.Lambda.Core;
+using Amazon.Lambda.Serialization.SystemTextJson.Converters;
+
+namespace Amazon.Lambda.Serialization.SystemTextJson
+{
+    /// <summary>
+    /// Custom ILambdaSerializer implementation which uses System.Text.Json
+    /// for serialization.
+    /// 
+    /// <para>
+    /// If the environment variable LAMBDA_NET_SERIALIZER_DEBUG is set to true the JSON coming
+    /// in from Lambda and being sent back to Lambda will be logged.
+    /// </para>
+    /// </summary>    
+    public class LambdaJsonSerializer : ILambdaSerializer
+    {
+        private const string DEBUG_ENVIRONMENT_VARIABLE_NAME = "LAMBDA_NET_SERIALIZER_DEBUG";
+        private readonly JsonSerializerOptions _options;
+        private readonly bool _debug;
+
+        /// <summary>
+        /// Constructs instance of serializer.
+        /// </summary>        
+        public LambdaJsonSerializer()
+        {
+            _options = new JsonSerializerOptions()
+            {
+                IgnoreNullValues = true,
+                PropertyNameCaseInsensitive = true,
```

Comment:
Are there more annotations that could be added that would make it possible to remove this? It comes with a performance penalty to some degree, so would be better if it could be left off.

Reply:
I was thinking users could use the constructor that takes in `Action<JsonSerializerOptions> customizer` to customize or remove these settings. I'm open to suggestions to make it easier.